lib/sshutils/marshal.go

```diff
@@ -40,20 +44,92 @@ func MarshalAuthorizedKeysFormat(clusterName string, keyBytes []byte) (string, e
 	return fmt.Sprintf("cert-authority %s %s", strings.TrimSpace(string(keyBytes)), comment.Encode()), nil
 }
 
-// MarshalAuthorizedHostsFormat returns the certificate authority public key exported as a single line
-// that can be placed in ~/.ssh/authorized_hosts. The format adheres to the man sshd (8)
-// authorized_hosts format, a space-separated list of: marker, hosts, key, and comment.
+// KnownHost is a structural representation of a known hosts entry for a Teleport host.
+type KnownHost struct {
+	AuthorizedKey []byte
+	ProxyHost     string
+	Hostname      string
+	Comment       map[string][]string
+}
+
+// MarshalKnownHost returns the certificate authority public key exported as a single line
```

Comment:
:nail_care: 
We can probably add a note saying that no new line is added to the string but might be required when adding the line to the file
It might get more confusing but it was something that I had to look up after reading the godoc.
Up to you :+1:

Reply:
I'll change the function to include the new line instead, which removes the need for some adhoc `\n` appending later on.